Traceback (most recent call last):
  File "/home/beyer/Dokumente/studienprojekt/gitlab_studienprojekt/fitting/paramfitt_singlePatient.py", line 338, in <module>
    patient_id = sys.arg[2]
AttributeError: module 'sys' has no attribute 'arg'. Did you mean: 'argv'?
Traceback (most recent call last):
  File "/home/beyer/Dokumente/studienprojekt/gitlab_studienprojekt/fitting/paramfitt_singlePatient.py", line 338, in <module>
    patient_id = sys.arg[2]
AttributeError: module 'sys' has no attribute 'arg'. Did you mean: 'argv'?
malloc_consolidate(): invalid chunk size
malloc_consolidate(): unaligned fastbin chunk detected
srun: error: golgo: task 0: Aborted
srun: error: golgo: task 1: Aborted
